docs/getting-started/example-deployment/additional-config.md

```diff
@@ -0,0 +1,107 @@
+# Additional Configuration
+
+This section will cover the smaller additional config files that go along side the main Gaffer
+schema to tweak other aspects of the Gaffer graph. The location of these files will need to be
+volume mounted into the container for them to be included in the deployment, the [next
+section](./running-the-deployment.md) on running the deployment discusses this in more detail.
+
+!!! note
+    Many of these files have defaults already in the standard `gaffer-rest` container image, but its
+    useful to still include them in the project to allow easy configuration.
+
+## Application Properties
+
+This is probably the simplest configuration file in the Gaffer deployment. In general it borrows a
+concept from [Spring
+Boot](https://www.tutorialspoint.com/spring_boot/spring_boot_application_properties.htm) to allow
+changing the context root and any properties related to Gaffer. In the example that follows we use
+it to set the file location properties of where the other config files are (inside the container).
+
+```properties title="application.properties"
+gaffer.schemas=/gaffer/schema
+gaffer.storeProperties=/gaffer/store/store.properties
+gaffer.graph.config=/gaffer/graph/graphConfig.json
+```
+
+## Graph Configuration
+
+The graph configuration file is a JSON file that configures few bits of the Gaffer graph. Primarily
+it is used to set the name and description along with any additional hooks to run before an operation
+chain e.g. to impose limits on max results etc. For the example as, it is a very basic graph we just
+set the name and short description.
+
+```json title="graphConfig.json"
+{
+    "graphId": "ExampleGraph",
+    "description": "An example graph"
+}
+```
+
+## Store Properties
+
+The store properties file is used to configure how Gaffer will store its data. There are a few
+different stores available for Gaffer, these are explained in more detail in the [reference
+documentation](../../reference/stores-guide/stores.md), but by default you must provide a store
+class and a store properties class. For this example we are using an Accumulo store as it is
```

Comment:
Giving an example of Accumulo is ok, but using Accumulo in docker as the later example adds complexity. In production Accumulo and Hadoop would never be used in this way. If we do use them, then we'd want to make that clear.

Reply:
I would argue using the containers more reduces complexity and brings it much more in line with industry standards that has moved to containerised mircroservices. Surely demonstrating it this way gives the user much more flexibility in where they run the cluster and you could re use the containers in k8s with suitable scaling for hdfs. I could add a disclaimer to say its probably not production example but we do need a recommended way to use Gaffer some where in the docs.

Reply:
Production HDFS is bare metal and on cloud uses a managed cluster provided as a service by the cloud platform. This PR is only an example deployment so it's fine to use, so long as it's clear that it's just an example. It does work in containers but this is only useful for simulating a cluster. My comment was around how it might be easier if we used the map store as the example. I should have put the comment in the right place as this isn't relevant to these example properties.

Reply:
That is true that generally hdfs is bare metal, but you can get it to run on cloud e.g. [uber did it](https://www.uber.com/en-GB/blog/hadoop-container-blog/) there is also apache ozone which I think is meant to work much better on deployments like that and HCFS alternatives but probably a wider discussion there. Fair comment, I was trying to not use the map store as that is essentially what I used for the quickstart guide, if I add few bits to explain its not a production ready example but demos the main deployment concepts that should hopefully clarify things here.